flutter/test/sentry_flutter_test.dart

```diff
@@ -53,7 +53,7 @@ void main() {
         packageLoader: loadTestPackage,
         platformChecker: getPlatformChecker(platform: MockPlatform.android()),
       );
-    });
+    }, testOn: 'vm');
```

Comment:
All these tests with `, testOn: 'vm'` doesn't need to be run on browser because they're for not browser targets.
Also they fail when run on browser.
I think it's okay to not run them on browser.

Reply:
maybe we delete `sentry_flutter_web_test.dart` if we dont need it?